README.md

```diff
@@ -16,94 +16,14 @@ Please note that the data for GSD is `CC0`, everything else is `Apache v2.0`. Se
-- **Depends on:**  GSD Database
-
-### GSD Bot
-
-- **Website:** _N/A_
-- **Location:** `gsd-tools/gsd-bot/`
-- **README:** [GSD Bot README](gsd-bot/README.md)
-- **Short Description:** Provides automation around specifically formatted issues within the gsd-database repo.
-- **Depends on:** _None_
-
-### GSD GitHub Actions
-
-- **Website:** _N/A_
-- **Location:** `gsd-tools/gsd-github-actions/`
-- **README:** [GSD GitHub Actions README](gsd-github-actions/README.md)
-- **Short Description:** The GitHub actions used by GSD Database to validate new and existing identifiers.
-- **Depends on:** _None_
-
-### GSD Requests
-
-- **Website:** https://requests.globalsecuritydatabase.org
-- **Location:** `gsd-tools/gsd-requests/`
-- **README:** [Requests Webform README](gsd-requests/README.md)
-- **Short Description:** A utilitarian webform for submitting new GSD IDs.
-- **Depends on:** GSD Database
-
-### GSD Schema
-
-- **Website:** _N/A_
-- **Location:** `gsd-tools/gsd-schema/`
-- **README:** [GSD Schema README](gsd-schema/README.md)
-- **Short Description:** Provides the json schema of a GSD identifier.
-- **Depends on:** _None_
-
-### GSD URL Processing
-
-- **Website:** _N/A_
-- **Location:** `gsd-tools/gsd-url-processing/`
-- **README:** [GSD URL Processing README](gsd-url-processing/README.md)
-- **Short Description:** Various scripts written by @kurtseifried to process urls into GSD ids.
-- **Depends on:** _None_
-
-### GSD Web
-
-- **Website:** https://gsd.id/home
-- **Location:** `gsd-tools/gsd-web`
-- **README:** [GSD Web README](gsd-web/README.md)
-- **Short Description:** A human interface for interacting with and updating the GSD data.
-- **Depends on:** GSD API, GSD Schema
-
-### Local Scripts
-
-- **Website:** _N/A_
-- **Location:** `gsd-tools/local-scripts/`
-- **README:** [Local Scripts README](local-scripts/README.md)
-- **Short Description:** _TODO_ - Ask @kurtseifried
-- **Depends on:** _None_
-
-### GSD Project Notes
-
-- **Website:** _N/A_
-- **Location:** `gsd-tools/gsd-project/`
-- **README:** [GSD URL Processing README](gsd-project/README.md)
-- **Short Description:** Project Plans which contains all the GSD project plans and related material. 
-- **Depends on:** _None_
+### Project Directory
+
+- [GSD Analysis](/gsd-analysis#gsd-analysis "A review of the GSD data using a python parser")
+- [GSD API](/gsd-api#gsd-api "REST API for interfacing with the GSD Database")
+- [GSD Bot](/gsd-bot#gsd-bot "Automation around specifically formatted issues within the gsd-database repo")
+- [GSD GitHub Actions](/gsd-github-actions#github-actions "GSD Github Actions")
+- [GSD Requests](/gsd-requests#webform "Webform for submitting new GSD IDs")
+- [GSD Schema](/gsd-schema#gsd-schema "JSON schema of a GSD identifier")
+- [GSD URL Processing](/gsd-url-processing#gsd-url-processing)
+- [GSD Web](/gsd-web#gsd-web "A web based user interface for the GSD data")
+- [Local Scripts](/local-scripts#local-scripts)
```

Comment:
This format provides the one-line description when you hover, but I suspect that might not be intuitive for someone viewing this for the first time. I do like the simplified format though.

Reply:
This isn't expressly for browser users. It's for screen readers and good indexing. If you think there should be a contextual description for these links to help describe them I don't mind putting them back.

Reply:
Ah, that makes sense.

It's fine for now, we can also add the descriptions back if folks mention it's confusing without them.